portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -143,8 +145,8 @@ private UserComment entityToComment(Entity entity) {
             UserComment.voteStatus.DOWNVOTED;
     }
 
-    UserComment userComment = UserComment.create(name, email, comment, time, id, parentId, rootId,
-        upvotes, downvotes, votingStatus);
+    UserComment userComment = UserComment.create(name, email, comment, time, id, parentId, rootId
+        , upvotes, downvotes, isEditable, votingStatus);
```

Comment:
Hmm Github seems to be reverting your prior edits on formatting... I wonder if it's because you edited this line in a prior PR but during a merge Github takes the edited version of the current client so the edit was effectively reverted?